src/host/ut_host/TextBufferTests.cpp

```diff
@@ -2065,7 +2065,7 @@ void TextBufferTests::GetWordBoundaries()
         { { 79, 0 },    {{ 10, 0 },      { 5, 0 }} },
 
         // tests for second line of text
-        { {  0, 1 },     {{ 0, 1 },       { 0, 1 }} },
+        { {  0, 1 },     {{ 0, 1 },       { 5, 0 }} },

```

Comment:
I'm not sure which of the code changes made this change. Is this good and righteous? It _seems_ like it is because a11y defines the word to be the word + all the spaces after it, so you're inside the spaces at 0,1... so the word starts at 5,0... but why was this broken before?

Reply:
This one was a mistake on my part back when I wrote these tests. You're right to be concerned but here's my reasoning:

You have the following text:
```
word other
  more   words
```

You're current position is where the X is (0,1):
```
word other
X more   words
```

When we get the previous word, we're supposed to wrap around to encompass "other". Before, on accident, we would hit the left boundary and _not_ wrap. So we would actually return the whitespace on the second line between the left boundary and "more".

Reply:
how did this test not _fail_ then? o_O

Reply:
The test didn't fail because I wrote it incorrectly. It used to expect the whitespace. It _should_ have expected the word on the previous line.

Reply:
I can't figure out how to permalink code, but you want to look at textbuffer.cpp line 1000
```c++
if (target.X == GetSize().Left())
```
We were treating "Left" as "Origin" and refusing to expand left.